conf/application.conf

```diff
@@ -86,7 +86,7 @@ mongodb {
 urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:10905/trader-goods-profiles"
-  signOut       = "http://localhost:9025/gg/sign-out"
+  signOut       = "http://localhost:9553/bas-gateway/sign-out-without-state"
```

Comment:
bas-gateway - typo?

Reply:
looks right to me

Reply:
It's an acronym :) https://confluence.tools.tax.service.gov.uk/display/MIN/BAS